cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1595,166 +1595,280 @@ action.summary_index.alert_name = Splunk Admin - Missing Forwarder
+| addinfo | where time>=info_min_time and time<=info_max_time \
+| deviceinventorygen
+
+
+# Backfill
+[Device Inventory Backfill]
+disabled = 0
+enableSched = 0
+alert.track = 0
+run_on_startup = true
+description = This report backfills the all device inventory lookups. (Default timerange: last 30 days). \
+The report executes on Splunk start to make sure we do not miss data as well as this make sure user do not have make manually tasks.
+dispatch.earliest_time = -30d@m
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = | savedsearch "Device Inventory - Tenable Vuln" \
+| append [| search `cs_lansweeper` `cs_lansweeper_timerange` | `cs_lansweeper_inventory_fill_search`] \
+| append [| savedsearch "Device Inventory - Tenable"] \
+| append [| savedsearch "Device Inventory - Qualys"] \
+| append [| savedsearch "Device Inventory - Sophos"] \
+| append [| search `cs_windows_defender` EventCode=1151 `cs_windows_defender_max_timerange` | `cs_windows_defender_inventory_fill_search`] \
+| append [| savedsearch "Device Inventory - CrowdStrike"] \
+| appendpipe [| where SEARCHNOTHING="SEARCHNOTHING" | append [| savedsearch "Device Inventory Gen"]]
+
+
+# CleanUp
+[Device Inventory Temp Lookup CleanUp]
+disabled = 0
+enableSched = 1
+alert.track = 0
+cron_schedule = 0 4 * * *
+description = This report cleans up temp device inventory lookup.
+dispatch.earliest_time = -7d@m
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = | inputlookup cs_device_inventory_temp \
+| addinfo | where indextime>=info_min_time and indextime<=info_max_time \
+| outputlookup cs_device_inventory_temp
+
+
+[Device Inventory Lookup CleanUp]
+disabled = 0
+enableSched = 1
+alert.track = 0
+cron_schedule = 0 5 * * 0
+description = This report cleans up device inventory lookup every week (sunday early morning) and removes the devices from the inventories which did not showup in last 60 days.
+dispatch.earliest_time = -60d@d
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = | inputlookup cs_device_inventory \
+| addinfo | where time>=info_min_time and time<=info_max_time \
+| outputlookup cs_device_inventory | where SEARCHNOTHING="SEARCHNOTHING" \
+| append [| inputlookup cs_device_inventory_merge_logs.csv \
+| addinfo | where indextime>=info_min_time and indextime<=info_max_time \
+| outputlookup cs_device_inventory_merge_logs.csv | where SEARCHNOTHING="SEARCHNOTHING"] \
+| append [| inputlookup cs_lansweeper_inventory \
```

Comment:
We don't need to cleanup for lansweeper.

Reply:
Removed